task3/update/index.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="logo.png">
+    <title> Ribbitter | Feed </title>
+</head>
+<body>
+<header>
+    <nav class="container">
+        <a class="logo" href="index.html">
+            <img src="logo.png" alt="A VERY MUCH GREEN FROG">
+            <span class="logotext"><h1>Ribbitter</h1></span>
```

Comment:
you don't need span here you can provide styles directly to to the h1
`<h1 class="logo">...</h1>`